build.gradle

```diff
@@ -1,16 +1,43 @@
 plugins {
-    id 'io.spring.dependency-management' version '1.0.3.RELEASE'
-    id 'org.springframework.boot' version '1.5.6.RELEASE'
+  id 'checkstyle'
+  id 'findbugs'
+  id 'org.sonarqube' version '2.5'
+  id 'jacoco'
+  id 'io.spring.dependency-management' version '1.0.3.RELEASE'
+  id 'org.springframework.boot' version '1.5.6.RELEASE'
+  id 'org.owasp.dependencycheck' version '1.4.5.1'
+  id 'com.github.ben-manes.versions' version '0.15.0'
 }
 
+group = 'uk.gov.hmcts.reform'
 version = '0.0.1'
+
 sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+checkstyle.toolVersion = '8.2'
+checkstyle.configFile = new File(rootDir, "checkstyle.xml")
```

Comment:
unless they fixed the issue you may need to add some more config to fail on warnings.
https://github.com/hmcts/cmc-claim-store/blob/master/build.gradle#L28

Reply:
done